libr/core/cmd_search.c

```diff
@@ -2079,6 +2056,9 @@ static void do_asm_search(RCore *core, struct search_parameters *param, const ch
 		input ++;
 	}
 	r_list_foreach (param->boundaries, itermap, map) {
+		if (!r_itv_overlap (search_itv, map->itv)) {
```

Comment:
not sure if we want to skip the whole interval just because it overlaps a portion of another map..